pkgs/development/tools/hindsight/default.nix

```
@@ -0,0 +1,70 @@
+{ lib, pythonPackages, fetchFromGitHub, nodejs, nodePackages }:
+
+with pythonPackages;
+
+buildPythonApplication rec {
```

```suggestion
{ lib, python3, fetchFromGitHub, nodejs, nodePackages }:

with python3.pkgs;

buildPythonApplication rec {
```

easier to track what is in the namespace if you only import other top-level attrs